org/allPRs.ts

```diff
@@ -86,12 +86,7 @@ export const localizedStrings = async () => {
         // * NSLocalizedString(
         // This way it will match `NSLocalizedString(` but not `NSLocalizedString` (without the opening parenthesis, which could be used in a comment).
         if (addedLines?.find(value => /\bNSLocalizedString\(/.test(value))) {
-            let instructions = "";
-            if (danger.github.thisPR.repo == "iOS") {
-                instructions = " See [Localization Guidelines](https://app.asana.com/0/0/1185863667140706/f) for more information.";
-            } else if (danger.github.thisPR.repo == "macos-browser") {
-                instructions = " See [Localization Guidelines](https://app.asana.com/0/0/1206727265537758/f) for more information.";
-            }
+            let instructions = " See [iOS](https://app.asana.com/0/0/1185863667140706/f) and [macOS](https://app.asana.com/0/0/1206727265537758/f) localization guidelines for more information.";
```

Comment:
I figure let's just combine these into a single warning rather than checking for modified paths.